modules/claims_api/lib/claims_api/v2/params_validation/intent_to_file/intent_to_file_validator.rb

```diff
@@ -11,13 +11,17 @@ def validate(record)
             validate_participant_vet_id(record)
           end
 
+          def self.itf_types
+            %w[compensation pension survivor].freeze
+          end
+
           private
 
           def validate_type(record)
             value = record.data[:type]
             (record.errors.add :type, 'blank') && return if value.blank?
 
-            record.errors.add :type, value unless %w[compensation pension].include?(value.downcase)
+            record.errors.add :type, value unless self.class.itf_types.include?(value.downcase)
```

Comment:
hmm. see comment: 
https://github.com/department-of-veterans-affairs/vets-api/pull/11411/files#r1056534625

I don't mind `self.itf_types` as much here but I still think we don't want this array in multiple spots. we'll have to switch both values if there ever were to be a change. I would rather run off something similar to the above.